### PR TITLE
Doc review of Event Type Inheritance

### PR DESCRIPTION
**Pull request by gladius-mtl**
Closes GH-1038

### CODE REVIEW

File: _pages/dev/event-service.md [hunk 1]
```diff
@@ -124,3 +133,60 @@ onClick() {
...
+
+## Event Type Inheritance
+
+{% capture version_note %}
+{{ site.version_note_part1a }} 3.1 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+Parent events, also called umbrella events, allow you to group similar events under one common event. The parent event can be an abstract class or a regular class. By subscribing to the parent event, you get emissions from all of the child events that inherit it.
```
Comment (znikola) [inline _pages/dev/event-service.md:145]: Maybe we change the wording here a bit, as we don't really want to call them umbrella events. We just used that expression to visually describe what the parent events could be useful for. Maybe we can even remove the term all together.

File: _pages/dev/event-service.md [hunk 2]
```diff
@@ -124,3 +133,60 @@ onClick() {
...
+
+For example, `PageEvent` from `@spartacus/storefrontlib` is a parent event, and all of the child page events, such as `HomePageEvent`, `CartPageEvent`, and `ProductDetailsPageEvent`, inherit from this parent. If you subscribe to `PageEvent`, you will get emissions from all of the child page events that inherit from `PageEvent`. The following is an example:
+
+```typescript
+eventService.get(PageEvent).subscribe(...) // receives all page events
+```
+
+All events should inherit from `CxEvent` in `@spartacus/core`, as shown in the following example:
```
Comment (znikola) [inline _pages/dev/event-service.md:153]: ..._from `CxEvent` in `@spartacus/core`_...
To me, this sounds like: all events that are created in `projects/core` lib should inherit from the `CxEvent`, while the rest of events (that are created in other libs) do not have to.

File: _pages/dev/event-service.md (hunk 1 above)
Comment (gladius-mtl) [reply _pages/dev/event-service.md:145]: Sounds good 👍

File: _pages/dev/event-service.md (hunk 2 above)
Comment (gladius-mtl) [reply _pages/dev/event-service.md:153]: Good point! I changed it to 

> All events should inherit from the `CxEvent` that is in `@spartacus/core`...

**Pull request merged**